backend/src/services/master/master.service.ts

```diff
@@ -60,6 +60,11 @@ class Master {
     };
   }
 
+  async getCurrentUser(token: string): Promise<MasterSignUpResponseDto> {
+    const { data } = this.#tokenService.decode(token);
```

Comment:
create this type in the `shared/common/types/token/`
```tsx
type TokenPayload = {
  userId: number;
};
```

```suggestion
    const { data } = this.#tokenService.decode<TokenPayload>(token);
```